Feat/workflow

Ryunos96 opened the pull request.

Ryunos96 committed:
feat: deploy.yml 파일 추가

Ryunos96 committed:
test: workflow test v1

Ryunos96 committed:
test: workflow test v2

Ryunos96 committed:
test: workflow test v3

Ryunos96 committed:
test: workflow test v4

Ryunos96 committed:
test: workflow test v5

Ryunos96 committed:
test: workflow test v6

Ryunos96 committed:
test: workflow test v7

Ryunos96 committed:
test: workflow test v8

Ryunos96 committed:
test: workflow test v9

Ryunos96 committed:
test: workflow test v10

Ryunos96 committed:
test: workflow test v11

Ryunos96 committed:
test: workflow test v11

Ryunos96 committed:
Remove: change workflow to develop and testfile delete

YunYongWoon commented on .github/workflows/deploy.yml:
```diff
@@ -0,0 +1,40 @@
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Build with Maven
```
github action 기능중에 .properties 파일을 복붙하는 방법이 있을꺼임. 인스턴스 환경변수에 등록하는것보다 깃헙액션에서 관리하는게 추후 배포 과정에서 관리가 더 용이할거라 생각합니다.

[관련링크](https://velog.io/@sun1203/Github-Action-Spring-Application.properties-%EB%AF%BC%EA%B0%90%EC%A0%95%EB%B3%B4-%EA%B4%80%EB%A6%AC%ED%95%98%EA%B8%B0)

YunYongWoon reviewed (approved):
내용 확인 한번만 체크하고 적용(optional) 후 머지 ㄱㄱ

Ryunos96 committed:
feat:deploy.yml파일 변경

application.properties github action으로 관리하는 기능 추가

Ryunos96 committed:
feat: deploy.yml파일 변경

secret 관리시 application.properties 인코딩없이 집어넣고 app.jar파일과 같은 위치에서 경로지정해서 적용되는 코드 추가

Ryunos96 committed:
test: deploy test

Ryunos96 committed:
feat: swagger 설정 변경

bean 어노테이션 추가

Ryunos96 committed:
feat: swagger 오류 해결

application.properties로 경로 설정

Ryunos96 committed:
test: swagger 에러 처리

Ryunos96 committed:
feat: feat/workflow -> develop (deploy 브랜치 변경)
